react/features/conference/components/web/Conference.js

```diff
@@ -217,6 +218,7 @@ class Conference extends Component<Props> {
                 id = 'videoconference_page'
                 onMouseMove = { this._onShowToolbar }>
                 <Notice />
```

Comment:
Will Notice and Subject overlap each other? Do you care?

Reply:
They are overlapping but the notice is fully visible. I guess it is OK for first version.